project/ProjectSettings.scala

```diff
@@ -22,7 +22,7 @@ object ProjectSettings {
     scalacOptions ++= Seq(
       "-unchecked",
       "-deprecation",
-      "-target:jvm-1.8",
+      "-release:11",
```

Comment:
`-target` [got deprecated in Scala 2.13.9](https://github.com/scala/scala/pull/9982). 

I am not entirely sure why we had `jvm-1.8` here. The minimum JVM version this code requires is 11 and `-release:8` throws errors. I might be misunderstanding the difference between the old `-target` and new `-release` though. If anyone knows more about this, an explanation would be much appreciated since I would like to understand this better!

Reply:
> I am not entirely sure why we had `jvm-1.8` here.

I think it was just missed in the various stages of adopting Java 11 (eg https://github.com/guardian/frontend/pull/24064 maybe).

Well done on using the colon in `"-release:11"` by the way! The PR description in https://github.com/scala/scala/pull/9982 seems to suggest it should be `"-release 11"`, but for that the scalac compiler says:

```
[warn] bad option '-release 11' was ignored
```

Reply:
either form is ok, but two tokens in the build file, `Seq("--release", "11")`. (I prefer the colon.)

Worth adding that `--target` affects the "classfile version" but scalac generally does not leverage JVM features where it matters. TIL a decade ago the setting mattered more to scalac.